.github/workflows/tlaplus.yml

```diff
@@ -114,3 +116,137 @@ jobs:
           path: |
             tla/consistency/*_TTrace_*.tla
             tla/*.json
+
+  model-checking-consensus:
+    name: Model Checking - Consensus
+    runs-on: [self-hosted, 1ES.Pool=gha-virtual-ccf-sub]
+    container:
+      image: ghcr.io/microsoft/ccf/ci/default:build-26-06-2024
+
+    steps:
+      - uses: actions/checkout@v4
+      - run: |
+          sudo apt update
+          sudo apt install -y default-jre
+          python3 ./tla/install_deps.py
+
+      - name: MCccfraft.cfg
```

Comment:
The model checking job is 1h47, I think it would be preferable to split it in three jobs so it only takes as long as the slowest step (WithReconfig). We should also try to disable the coverage, that lead to massive speedups in the past.

Reply:
Yes, coverage collection incurs a significant slowdown.